Numerical_solution_of_SEIRD.Rmd

```diff
@@ -0,0 +1,356 @@
+---
+title: "Numerical solution of SEIRD"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Numerical solution of SEIRD}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+editor_options:
+  markdown:
+    wrap: 72
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+library(comomodels)
+library(deSolve)
+library(ggplot2)
+library(tidyverse)
+library(glue)
+```
+
+## Introduction
+
+Compartmental epidemiological models (such as SEIR and its variants) are 
+_differential equation_ models—that is, given particular values of the 
+model parameters, they do not directly specify the output of the model over
+time, but rather they specify a system of differential equations whose solution
+is the model output over time. Like many differential equation models of
+interest in scientific applications, SEIR differential equations do not admit
+analytical solutions. Instead, an approximation of the solution must be
+obtained using numerical methods.
+
+Epidemiological differential equations can be solved easily using simple fixed
+time step solver such as the Euler method. In this notebook, we show that when
+such methods are employed without sufficient care in the selection of the time
+step, unacceptably high error in the numerical solution may occur. We recommend
```

Comment:
"unacceptably" sounds quite strong? Not sure how to define that, so maybe take it out.